src/architecture/serialization.md

```diff
@@ -1,533 +1,193 @@
-# Exonum Serialization Format
+# Serialization in Exonum
 
 <!-- cspell:ignore cap'n -->
 
-**Binary serialization format** is used in Exonum for communication
-among full nodes, cryptographic operations on [light clients](../architecture/clients.md)
-and storage of data. The format design provides several important properties,
-including resilience to maliciously crafted messages, [zero-copy][zero_copy]
-deserialization and canonicity.
-
-Serialization in Exonum differs from serialization in the usual sense, since
-there is no process of transforming the structure into binary data. The data is
-created already "serialized" and Exonum works directly with the serialized data
-"deserializing" the fields to which it refers, if necessary.
+Exonum uses [protobuf][protobuf] as its serialization format for communication
+among full nodes, cryptographic operations on
+[light clients](../architecture/clients.md)
+and storage of data. Protobuf is the industry accepted language-neutral and
+platform-neutral automated mechanism for serializing data.
 
 ## Usage
 
-### Communication Among Full Nodes
-
-Full nodes can both [serialize messages](#message-serialization) for sending and
+**Communication Among Full Nodes** Full nodes can both
```

Comment:
Nit: I like the previous formatting with headers better, but I suppose this one makes sense too.